doctest/parts/doctest_fwd.h

```diff
@@ -862,10 +870,8 @@ namespace detail {
     template<class T>
     using has_insertion_operator = has_insertion_operator_impl::check<const T>;
 
-    DOCTEST_INTERFACE void my_memcpy(void* dest, const void* src, unsigned num);
-
-    DOCTEST_INTERFACE std::ostream* getTlsOss(bool reset=true); // returns a thread-local ostringstream
-    DOCTEST_INTERFACE String getTlsOssResult();
+    DOCTEST_INTERFACE std::ostream& tlssPush();
```

Comment:
I still haven't processed the entire changeset, but 1 quick comment would be that some MSVC version couldn't compile when a reference was used instead of a pointer (in case you were wondering), even though the reference was just passed around and no method was being called... But the builds seem to pass so I guess it's time to switch back to a ref.

Reply:
Actually MSVC 2015 doesn't seem to be included in the github actions build matrix - it was being tested on appveyor until that worked but I didn't set it up properly now that doctest moved to an org... so the latest version was released without testing 2015...

Reply:
Oh damn, that really sounds suboptimal. So I think the best course of action for this PR is to wait for the dev branch to be fixed and then rerun the builds on this?

Reply:
yeah... also this issue:
https://github.com/doctest/doctest/issues/568

I'll be looking closer at commit 02af672f0d3093157daecb55ce4e109ffaf2bc53